test/integration/basic_dataplane_standalone_test.go

```diff
@@ -128,6 +128,8 @@ type standaloneDataplaneFixture struct {
 
 func (fixture *standaloneDataplaneFixture) cleanup() {
 	fixture.forwarding.Stop()
+	// Let's delete source/destPod without gracetimeout
+	fixture.k8s.DeletePodsForce(fixture.sourcePod, fixture.destPod)
```

Comment:
Nope, actually this is not required anymore, since alphine pods are set with gracefull timeout defined to 0. But in case it will be changed. And this is only for dataplane tests  source/destination pods.